AssertionError in test_load_image

```
[2024-12-28T19:19:03.755Z] ======================================================================

[2024-12-28T19:19:03.755Z] FAIL: test_nibabel_reader_gpu_3 (tests.test_load_image.TestLoadImage)

[2024-12-28T19:19:03.755Z] ----------------------------------------------------------------------

[2024-12-28T19:19:03.755Z] Traceback (most recent call last):

[2024-12-28T19:19:03.755Z]   File "/usr/local/lib/python3.10/dist-packages/parameterized/parameterized.py", line 620, in standalone_func

[2024-12-28T19:19:03.755Z]     return func(*(a + p.args), **p.kwargs, **kw)

[2024-12-28T19:19:03.755Z]   File "/home/jenkins/agent/workspace/Monai-latest-image/tests/test_load_image.py", line 236, in test_nibabel_reader_gpu

[2024-12-28T19:19:03.755Z]     self.assertTrue(torch.equal(result_cpu, result.cpu()))

[2024-12-28T19:19:03.755Z] AssertionError: False is not true
```

--- a/tests/test_load_image.py
+++ b/tests/test_load_image.py
@@ -233,7 +233,7 @@ class TestLoadImage(unittest.TestCase):
             input_param_cpu = input_param.copy()
             input_param_cpu["to_gpu"] = False
             result_cpu = LoadImage(image_only=True, **input_param_cpu)(filenames)
-            self.assertTrue(torch.equal(result_cpu, result.cpu()))
+            self.assertTrue(torch.allclose(result_cpu, result.cpu(), atol=1e-6))
 
     @parameterized.expand([TEST_CASE_6, TEST_CASE_7, TEST_CASE_8, TEST_CASE_8_1, TEST_CASE_9])
     def test_itk_reader(self, input_param, filenames, expected_shape):

